test/Cis194/Hw/Week1Spec.hs

```diff
@@ -2,8 +2,8 @@ module Cis194.Hw.Week1Spec (main, spec) where
 
 import Test.Hspec
 import Test.QuickCheck
-import Cis194.Hw.Week1
-{-import Cis194.Hw.AcceptHanoi-}
+import Cis194.Hw.Week1 hiding (main)
```

Comment:
I didn't know about this `hiding` thing. I'm not sure what it is doing. I thought it would hide a function, but I don't see a main function in the module.

Reply:
Whoops, that's a hold-over which should be removed. I _did_ have a `main` function to run some of my own "tests" before I figured out how to run this spec file. `hiding` was excluding that main function from the import because it conflicted with the `main` function in this spec module.